awsrds/rds_db_instance_test.go

```diff
@@ -201,6 +201,104 @@ var _ = Describe("RDS DB Instance", func() {
 		})
 	})
 
+	var _ = Describe("GetTag", func() {
+		var (
+			properDBInstanceDetails DBInstanceDetails
+
+			describeDBInstances []*rds.DBInstance
+			describeDBInstance  *rds.DBInstance
+
+			describeDBInstancesInput *rds.DescribeDBInstancesInput
+			describeDBInstanceError  error
+			expectedTag              string
+		)
+
+		BeforeEach(func() {
+			properDBInstanceDetails = DBInstanceDetails{
+				Identifier:       dbInstanceIdentifier,
+				Status:           "available",
+				Engine:           "test-engine",
+				EngineVersion:    "1.2.3",
+				DBName:           "test-dbname",
+				MasterUsername:   "test-master-username",
+				AllocatedStorage: int64(100),
+				Tags: map[string]string{
+					"SkipFinalsnapshot": "true",
+				},
+			}
+
+			describeDBInstance = &rds.DBInstance{
+				DBInstanceIdentifier: aws.String(dbInstanceIdentifier),
+				DBInstanceStatus:     aws.String("available"),
+				Engine:               aws.String("test-engine"),
+				EngineVersion:        aws.String("1.2.3"),
+				DBName:               aws.String("test-dbname"),
+				MasterUsername:       aws.String("test-master-username"),
+				AllocatedStorage:     aws.Int64(100),
+			}
+			describeDBInstances = []*rds.DBInstance{describeDBInstance}
+
+			describeDBInstancesInput = &rds.DescribeDBInstancesInput{
+				DBInstanceIdentifier: aws.String(dbInstanceIdentifier),
+			}
+			describeDBInstanceError = nil
+			expectedTag = "true"
+		})
+
+		JustBeforeEach(func() {
+			rdssvc.Handlers.Clear()
+
+			rdsCall = func(r *request.Request) {
+				switch r.Operation.Name {
+				case "DescribeDBInstances":
+					Expect(r.Operation.Name).To(Equal("DescribeDBInstances"))
```

Comment:
is this a meaningful test? In switch r.Operation.Name and in case "DescribeDBInstances" you test if r.Operation.Name is really "DescribeDBInstances"? Is this supposed to test _switch_ and _case_ functionality?